src/transformers/models/grounding_dino/processing_grounding_dino.py

```diff
@@ -60,6 +65,57 @@ def get_phrases_from_posmap(posmaps, input_ids):
     return token_ids
 
 
+def _is_merged_candidate_labels_text(text: str) -> bool:
+    """
+    Check if the text is a merged candidate labels text. The general rule for constructing
+    candidate labels string is to separate the text labels with a dot. For example, "cat. dog."
+    """
+    if isinstance(text, str) and "." in text:
+        return True
+    return False
```

Comment:
I don't really think we need this one liner!

Reply:
Removed this one